core/types/receipt.go

```diff
@@ -310,6 +310,8 @@ func (r Receipts) DeriveFields(config *params.ChainConfig, hash common.Hash, num
 		r[i].BlockNumber = new(big.Int).SetUint64(number)
 		r[i].TransactionIndex = uint(i)
 
+		r[i].Bloom = CreateBloom(Receipts{r[i]})
```

Comment:
Aren't the blooms already handled during decoding?

Reply:
As far as I see it those are already covered.

Reply:
@karalabe You are right -- my bad. I hadn't had my morning  ☕️ yet.

This does raise the question: where should we be computing the bloom? It's kind of weird to be computing these fields in different places, but at the same time the bloom filter is immediately computable when decoding and the overhead is negligible. What would you prefer?